examples/visualization/plot_publication_figure.py

```diff
@@ -69,6 +69,7 @@
 # 255, 255)`` encodes a white pixel, so we'll detect any pixels that differ
 # from that:
 
+screenshot[(screenshot == 0)] = 255
```

Comment:
Just the alpha channel, right?
```suggestion
screenshot[..., 3][screenshot[..., 3] == 0] = 255
```